understanding/20/keyboard-no-exception.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Keyboard (No Exception)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Everything can be done with a keyboard</dd>
-         <dt>Author task</dt><dd>Ensure all pointer actions have a keyboard equivalent</dd>
+         <dt>Goal</dt><dd>Everything can be done with a keyboard.</dd>
+         <dt>What to do</dt><dd>Ensure all pointer actions have a keyboard equivalent.</dd>
+         <dt>Why it's important</dt><dd>People who can only use the keyboard need to be able to accomplish everything.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People who can only use the keyboard or keyboard alternatives (speech, Morse code, scanning, etc.) need to be able to accomplish everything from the keyboard interface.</dd> 
```

Reply:
I think this is too much for a brief section, as per prior approach, I'm suggesting that since we already have "keyboard" called out, we can add "keyboard interface" here for a broader idea, which is then supported in the Understanding document.
```suggestion
         <dt>Why it's important</dt><dd>People who can only use the keyboard interface need to be able to accomplish everything.</dd> 
```

Reply:
It think this is ok if you add to the end.   "from a keyboard or alternate keyboard". 

so it reads.
  <dt>Why it's important</dt><dd>People who can only use the keyboard interface need to be able to accomplish everything from a keyboard or alternate keyboard.</dd> 

otherwise it seems circular and doesnt really convey the key information.   Everyone thinks it is just keyboard and that is a problem.  this fixes it .